src/twisted/topfiles/8764.feature

```diff
@@ -0,0 +1 @@
+Twisted is now versioned based on a year.month.patch basis.
```

Comment:
The major improvement is that you can now set version constraints, because you can predict exactly when the next incompatible version will be.